antlr/SchemaParser.ts

```
@@ -643,513 +773,1478 @@ export class SchemaParser extends Parser {
 	}
 }
 
 
-export class TypeDefinitionContext extends ParserRuleContext {
+export class IdentifierContext extends ParserRuleContext {
 	public IDENTIFIER(): TerminalNode { return this.getToken(SchemaParser.IDENTIFIER, 0); }
-	public fieldList(): FieldListContext {
-		return this.getRuleContext(0, FieldListContext);
+	constructor(parent: ParserRuleContext | undefined, invokingState: number) {
+		super(parent, invokingState);
+	}
```

Biome

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)